rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -1098,4 +1098,21 @@ extern "C" RocalTensor ROCAL_API_CALL rocalSSDRandomCrop(RocalContext context, R
                                                          RocalTensorLayout output_layout = ROCAL_NONE,
                                                          RocalTensorOutputType output_datatype = ROCAL_UINT8);
 
+/*! \brief Applies preemphasis filter to the input data.
+ * \ingroup group_rocal_augmentations
+ * \param [in] context Rocal context
```

Comment:
Change to p_context

Reply:
For all the other augmentations in the file - context is defined this way.
* \param [in] context Rocal context

Reply:
I feel we should stick to context in this file . @fiona-gladwin Please let me know if thats fine